Detectors/MUON/MID/Simulation/src/Geometry.h

```diff
@@ -25,9 +25,15 @@ namespace o2
 namespace mid
 {
 
+enum RPCtype { Long,
```

Comment:
Would use a "enum class RPCtype" instead.

Reply:
I just changed it